.docker/qgis/scripts/qgis_testrunner.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
```

Comment:
This file you don't need to add, is already part of the docker image, see /usr/bin/qgis_testrunner.py

Reply:
I want not aware of this. I have removed this file